cmake/modules/LXQtCompilerSettings.cmake

```diff
@@ -179,16 +179,13 @@ endif()
 
 
 #-----------------------------------------------------------------------------
-# CXX11 and CXX0X requirements
-#-----------------------------------------------------------------------------
-CHECK_CXX_COMPILER_FLAG("-std=c++11" COMPILER_SUPPORTS_CXX11)
-CHECK_CXX_COMPILER_FLAG("-std=c++0x" COMPILER_SUPPORTS_CXX0X)
-if(COMPILER_SUPPORTS_CXX11)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
-elseif(COMPILER_SUPPORTS_CXX0X)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+# CXX14 requirements
```

Comment:
We should mess with the compiler flags by our self. Up from cmake 3.1 there is a `CXX_STANDARD` target property and the default `CMAKE_CXX_STANDARD`. We should use it together with `CXX_STANDARD_REQUIRED` (and the default `CMAKE_CXX_STANDARD_REQUIRED`).

ref. https://cmake.org/cmake/help/v3.1/prop_tgt/CXX_STANDARD.html#prop_tgt:CXX_STANDARD

Reply:
Agreed.